pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -260,6 +296,29 @@
 				</buttonbar>
 			</panel>
 
+			<tab><translate>Experimental</translate></tab>
```

Comment:
Maybe rename the tab to “Renderer”? Experimental looks too generic, there are other experimental options not being listed there (and it's better this way), and those options will be kept once the material system is not considered experimental anymore (we can't require bindless texture so the alternate code will remain).

Reply:
Hmm, I guess that makes sense. Changed to Renderer now.